dds/idl/annotations.h

```diff
@@ -302,66 +302,66 @@ namespace OpenDDS {
       xcdr1 |= other.xcdr1;
       xcdr2 |= other.xcdr2;
       xml |= other.xml;
-      unaligned_cdr |= other.unaligned_cdr;
+      //unaligned_cdr |= other.unaligned_cdr;
     }
 
     void set_all(bool value)
     {
       xcdr1 = value;
       xcdr2 = value;
       xml = value;
-      unaligned_cdr = value;
+      //unaligned_cdr = value;
     }
 
```

Comment:
I think all the methods below `set_all` could be actually be removed.

Reply:
I saw some other code calling `not_only_xcdr2` and `not_only_xcdr1`, so I'm keeping them for now.